cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -507,10 +507,15 @@ private static File updateMapredConfFile(File confFile, File tempDir) {
     String mrAppClassPath = conf.get(MRJobConfig.MAPREDUCE_APPLICATION_CLASSPATH,
                                      MRJobConfig.DEFAULT_MAPREDUCE_APPLICATION_CLASSPATH);
 
-    // Add the pwd/* at the beginning of classpath. Without this change, old jars from mr framework classpath
-    // get into classpath.
-    mrAppClassPath = "$PWD/*," + mrAppClassPath;
+    // We first add cdap-common.jar to the classpath so that FileContextLocationFactory from cdap-common gets
```

Comment:
we won't need this when ``FileContextLocationFactory`` is fixed in Twill, right? If so, create a jira for removing this?